src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -264,8 +264,14 @@ def _register_default_ciphers(self) -> None:
         self.register_cipher_adapter(
             TripleDES, ECB, GetCipherByName("des-ede3")
         )
+        # ChaCha20 uses the Short Name "chacha20" in OpenSSL, but in LibreSSL
+        # it uses "chacha"
```

Comment:
Nit: this is the "Long Name", actually :)

The [Short Name][1] is the capitalized "ChaCha20", or "ChaCha" respectively.

Reply:
Ah good catch, fixed here: https://github.com/pyca/cryptography/pull/9764